DA/submissions/pickled favorites

```diff
@@ -0,0 +1,9 @@
+import pickle
+favorites = ['football', 'wrestling', 'weightlifting', 'coding']
```

Comment:
So let's add a comment here telling us what the problem is and what we hope to achieve in this code

Reply:
done